### ISSUE TITLE
Null body in outbox - MSSQL

### ISSUE THREAD

**Issue by preardon** [BrighterCommand/Brighter#2235]
### Describe the bug
When trying to clear a message in the MSSQL outbox and it has a null body, an exception is thrown
When trying to insert a message with a null body, the MSSQL outbox allows you

**Comment by preardon** [comment]
```
System.Data.SqlTypes.SqlNullValueException:
   at Microsoft.Data.SqlClient.SqlBuffer.ThrowIfNull (Microsoft.Data.SqlClient, Version=4.1.0.0, Culture=neutral, PublicKeyToken=23ec7fc2d6eaa4a5)
   at Microsoft.Data.SqlClient.SqlBuffer.get_String (Microsoft.Data.SqlClient, Version=4.1.0.0, Culture=neutral, PublicKeyToken=23ec7fc2d6eaa4a5)
   at Microsoft.Data.SqlClient.SqlDataReader.GetString (Microsoft.Data.SqlClient, Version=4.1.0.0, Culture=neutral, PublicKeyToken=23ec7fc2d6eaa4a5)
   at Paramore.Brighter.Outbox.MsSql.MsSqlOutbox.MapAMessage (Paramore.Brighter.Outbox.MsSql, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b90f5623ae237e2c: /_/src/Paramore.Brighter.Outbox.MsSql/MsSqlOutbox.cs:704)
   at Paramore.Brighter.Outbox.MsSql.MsSqlOutbox+<OutstandingMessagesAsync>d__23.MoveNext (Paramore.Brighter.Outbox.MsSql, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b90f5623ae237e2c: /_/src/Paramore.Brighter.Outbox.MsSql/MsSqlOutbox.cs:489)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult (System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Paramore.Brighter.ExternalBusServices+<BackgroundDispatchUsingAsync>d__37.MoveNext (Paramore.Brighter, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b90f5623ae237e2c: /_/src/Paramore.Brighter/ExternalBusServices.cs:238)

```

![image](https://user-images.githubusercontent.com/7712876/185739522-65f10394-3a4c-4c60-b2bc-107acb6a44fb.png)

**Comment by preardon** [comment]
The fix has been released in 9.2.4
